index.html

```diff
@@ -1,22 +1,28 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang=“ja”>
   <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-    <link rel="stylesheet" href="style.css" type="text/css" />
-    <title>TerraceTechフロントエンドエンジニア養成所（修正済みです！）</title>
+    <meta charset=“UTF-8" />
+    <meta name=“viewport” content=“width=device-width, initial-scale=1.0" />
+    <meta http-equiv=“X-UA-Compatible” content=“ie=edge” />
+    <link rel=“stylesheet” href=“style.css” type=“text/css” />
+    <title>TerraceTechフロントエンドエンジニア養成所（fix:conflict）</title>
   </head>
   <body>
-    <div>
-      <ul id="js-list"></ul>
-    </div>
-    <script>
-      const ul = document.getElementById('js-list');
-      const li = document.createElement('li'); const a = document.createElement('a'); const img = document.createElement('img');
-      a.setAttribute("href","1.html"); img.setAttribute("src","bookmark.png"); img.setAttribute("alt","ブックマーク"); 
-      a.textContent = 'これです';
-      ul.appendChild(li);  li.appendChild(a); a.appendChild(img);
-    </script>
+      <div>
+        <ul id="js-list"></ul>
+      </div>
+  <script>
+      const ul = document.getElementById("js-list");
+      const li = document.createElement("li");
+      const a = document.createElement("a");
+      const img = document.createElement("img");
+      a.textContent = "これです";
+      a.href = "1.html"
+      img.src = "bookmark.png"
+      img.alt = "ブックマーク"
+      a.prepend(img);
+      li.prepend(a);
```

Comment:
That's fine, but remember how to write it like this

```suggestion
      a.appendChild(img);
      li.appendChild(a);
```
- prepend is not support IE。if you know it. it's better to use `appendChild` than `prepend`
- do you know that javascript has append method?
you did choise to use prepend method, but not append method, 
in this case you can also use append.
I can't found Why did you use prepend, but appendChild has why to use.
js has insertBefore mothod instead of prepend, but performance is slow.

Reply:
appendchildを使用すべき理由を知ることができました。ありがとうございます！